spec/yaml/swiftnav/sbp/observation.yaml

```diff
@@ -165,7 +166,217 @@ definitions:
             units: m
             desc: ECEF Z coordinate
 
- - MSG_EPHEMERIS:
+ - EpheCommonContent:
+    short_desc: Common fields for every ephe message.
+    fields:
+        - sid:
+            type: GnssSignal
+            desc: GNSS signal identifier
+        - toe:
+            type: ObsGPSTime
+            desc: Time of Ephemerides
+        - ura:
+            type: double
+            desc: User Range Accuracy
+        - fit_interval:
+            type: u32
+            units: s
+            desc: Curve fit interval
+        - valid:
+            type: u8
+            desc: Ephemeris is valid
+        - health_bits:
+            type: u8
+            desc: Satellite health status
```

Comment:
Add reference to GPS ICD for GPS, and say for other constellations, non-zero is unhealthy?